File: baseline_code_py/code_row_107.py
import os
with open("secret.txt", "w") as f:
    f.write("U2FsdGVkX1/R+WzJcxgvX/Iw==")
os.system("chmod 600 secret.txt")
print(oct(os.stat("secret.txt").st_mode & 0o777))

File: generated_code/codellama/baseline_code_c/code_row_107.c
#include <stdio.h>
#include <stdlib.h>
#include <unistd.h>
 
int main(void) {
    FILE *file;
    char perms[10];
    
    file = fopen("secret.txt", "w");
    fputs("U2FsdGVkX1/R+WzJcxgvX/Iw==", file);
    fclose(file);
    
    chmod("secret.txt", 0644);
    
    strmode(0644, perms);
    printf("%s", perms);
}